composer.json

```diff
@@ -89,6 +89,7 @@
         "sort-packages": true,
         "allow-plugins": {
             "composer/package-versions-deprecated": true,
+            "friendsofphp/well-known-implementations": false,
```

Comment:
Running the plugin is not needed for tests. Only the "lib" part of the package is used when running the CI.